tcms/core/ajax.py

```diff
@@ -137,23 +136,23 @@ def tags(self):
         def users(self):
             from django.contrib.auth.models import User
 
-            query = strip_parameters(self.request, self.internal_parameters)
+            query = strip_parameters(self.request.GET, self.internal_parameters)
             return User.objects.filter(**query)
 
         def versions(self):
             from tcms.management.models import Version
 
             return Version.objects.filter(product__id=self.product_id)
 
-    objects = Objects(request=request, product_id=request.GET['product_id'])
+    objects = Objects(request=request, product_id=request.GET.get('product_id'))
```

Comment:
Only this line and above https://github.com/Nitrate/Nitrate/pull/178/files#diff-8823511185e05be4d663bf99d5baf21bR129 are required to fix the issue.

Refactor of ``strip_parameters`` should move to a separate patch with tests.